config/core/configmaps/autoscaler.yaml

```diff
@@ -56,7 +73,8 @@ data:
     # NOTE: that this value does not affect actual number of concurrent requests
     #       the user container may receive, but only the average number of requests
     #       that the revision pods will receive.
-    container-concurrency-target-percentage: "70"
+    # This config will be deprecated, use target-utilization instead.
+    container-concurrency-target-percentage: "0"
```

Comment:
nit: I think it's worth mentioning in here that a `0` value will disable this option.

Reply:
Done.